docs: add 4 blog of APISIX Summit (#1151)
iresty/iresty.github.io: website/blog/2022/06/14/xueqiu-with-apache-apisix.md

---

> title: "Apache APISIX Helps Xueqiu Architecture Evolution"

Reviewer: ```suggestion
title: "Apache APISIX helps Xueqiu architecture evolution"
```

Reviewer: Here are some `Title Style` for reference 😄 

https://www.hashicorp.com/case-studies

authors:
  - name: "Xueqiu Basic Component Team"
    title: "Author"
  - name: "Sylvia"
    title: "Technical Writer"
    url: "https://github.com/SylviaBABY"
    image_url: "https://avatars.githubusercontent.com/u/39793568?v=4"
keywords: 
- API Gateway
- Apache APISIX
- Xueqiu
- Authentication
- Community
description: This article is compiled from the sharing at Apache APISIX Summit ASIA 2022, and introduces how Xueqiu uses APISIX to achieve more flexible services in the adjustment of the active-active architecture.
tags: [User Case]
---

> > This article is compiled from the sharing at Apache APISIX Summit ASIA 2022, and introduces how Xueqiu uses APISIX to achieve more flexible services in the adjustment of the active-active architecture.

Reviewer: ```suggestion
> This article is compiled from the sharing at [Apache APISIX Summit ASIA 2022](https://apisix-summit.org/). It introduces how Xueqiu uses APISIX to achieve more flexible services in adjusting the active-active architecture.
```
